templates/site/about.html

```diff
@@ -100,7 +100,57 @@ <h5>{{ row["title"] }}</h5>
         {% set title = "Acknowledgments" %}
         {% set intro_content = "The Marble climate platform was made possible with funding from the following agencies: " %}
         {% set content %}
-            <p>Acknowledgements are coming soon!</p>
+
+            {% set funder_images = [
+                {"image1": "/images/logos/BCID_Supported_H_RGB_pos.png",
+                "image2": "/images/logos/ontario-researh-fund.jpg",
+                "image3":"/images/logos/MEIE_-Avril_2014-300x130.jpg"
+                },
+                {"image1": "/images/logos/CFI-Logo-RGB.svg",
+                "image2": "",
+                "image3": ""
+                }
+            ] %}
+
+            {% set partner_images = [
+
+                {"image1": "/images/logos/ouranos_logo_horizontal_couleur.png",
+                "image2": "/images/logos/logo-CRIM-color-300x61.png",
+                "image3":"/images/logos/logo-pcic-long.jpg"
+                },
+                {"image1": "/images/logos/logo-University-of-toronto.png",
+                "image2": "/images/logos/logo-McGill-University.png",
+                "image3": "/images/logos/logo-Concordia.jpeg",
+                },
+                {"image1": "/images/logos/logo-Environment-and-Climate-Change-Canada.svg",
+                "image2": "/images/logos/logo-Natural-Resources-Canada.svg",
+                "image3": "",
+                }
+
+            ] %}
+
+
+
+            <h4 class="mt-2">Our Funders</h4>
+            {% for row in funder_images %}
```

Comment:
As discussed in the meeting, can we use a flexbox with the [flex-wrap](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flex-wrap) property so that the images scale rearrange nicely as the page width adjusts?

Reply:
Or just put them all into one image.

I would prefer the flexbox option if that option looks good. I consider putting everything into a single image as a good second option only.